Resources/Textures/Shaders/circle_mask.swsl

```diff
@@ -1,17 +1,21 @@
+uniform highp vec2 ZOOM_SCALE;
+
 highp vec4 circle(in highp vec2 uv, in highp vec2 pos, highp float rad, in highp vec3 color) {
     highp float d = length(pos - uv) - rad;
     highp float t = clamp(d, 0.0, 1.0);
     return vec4(color, t);
 }
 void fragment(){
 	highp vec2 uv = FRAGCOORD.xy;
+    highp vec2 uv_stretch = vec2(uv.x * ZOOM_SCALE.x, uv.y * ZOOM_SCALE.y);
```

Comment:
you should be able to just multiply vectors directly here and elsewhere
I.e., just use `= uv * ZOOM_SCALE;`